pkg/controller/importconfig/cluster_info.go

```diff
@@ -482,3 +486,62 @@ func getBootstrapSAName(clusterName string) string {
 	}
 	return bootstrapSAName
 }
+
+func filesToBytes(files ...string) []byte {
+	byteBuffer := new(bytes.Buffer)
+	for _, file := range files {
+		b, err := manifestFiles.ReadFile(file)
+		if err != nil {
+			// this should not happen, if happened, panic here
+			panic(err)
+		}
+		byteBuffer.WriteString(fmt.Sprintf("%s%s", constants.YamlSperator, string(b)))
+	}
+	return byteBuffer.Bytes()
+}
+
+func filesToTemplateBytes(files []string, config interface{}) []byte {
+	manifests := new(bytes.Buffer)
+	for _, file := range files {
+		template, err := manifestFiles.ReadFile(file)
+		if err != nil {
+			panic(err)
```

Comment:
why panic but not return error?

Reply:
The same reason as the above one, fixed.